### DISCUSSION TITLE
Layout das CENAS M³

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#357] [General]
Suggest
Taking advantage of this, I would like to suggest the layout of the SCENES similar to the image below, it's not just a matter of aesthetics, it's to make it easier to visualize the scene better. Thankful for your work.

![image](https://user-images.githubusercontent.com/90984283/169436055-46cf5cdf-6b99-4953-bff8-ff445845ecb7.png)

**Comment by sircharlo** [comment]
Good point! Implemented [here](https://github.com/sircharlo/meeting-media-manager/commit/be43b422afbd4e3bd0b2ebb176015f49e3a9b8aa).
